README.md

```diff
@@ -67,6 +67,21 @@ where:
 
 _This method calls `group.add` and `group.addAuthors` for you (adding you)_
 
+### `ssb.tribes.application.create(groupdId, text, cb)`
```

Comment:
this is missing the `authorIds` field that's present in the method. I think this could be an optional arg for the moment

Reply:
Correct. Added it.